javascripts/govuk/analytics/google-analytics-universal-tracker.js

```diff
@@ -160,6 +167,11 @@
     }
   }
 
+  function stripPIIFromString (string) {
+    var stripped = string.replace(EMAIL_PATTERN, '[email]')
```

Comment:
I wonder if the `EMAIL_PATTERN` variable should be in this function so it's closer to the code where it gets used and then maybe rename this to `stripEmailAddressesFromString`?